### ISSUE TITLE
Harmonise labels in tagpacks

### ISSUE THREAD

**Issue by wincentbalin** [INTERPOL-Innovation-Centre/TagPackConverters#24]
Currently, in many tagpack generators the data from raw data is converted to labels, providing too much information. For example, if ZEC address can be traced to PipeFlare, the investigator does not really need to know, when the address was used first (and if she does, she can look it up in any ZEC explorer).

This applies to many other tagpack generators.

**Comment by wincentbalin** [comment]
Add `abuse: extremism` to the SPLC-CR tagpack entries.

**Comment by VinceICPO** [comment]
Agreed, as long as we provide sufficient information for the investigator to cross-check the attribution tag source, we are good.

**Comment by wincentbalin** [comment]
This issue is not solved yet, as tagpack generators other than for PipeFlare also contain too verbose labels. Thus I would like this issue to reopened.
